babybear/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -0,0 +1,259 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: RobotContainer.java
+// Intent: Forms key glue class to land subsystems and input devices that will themselves create commands applied to the robot.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.DefaultDriveCommand;
+import frc.robot.commands.DriveTimeCommand;
+import frc.robot.commands.EveryBotPickerDefaultCommand;
+import frc.robot.commands.EveryBotPickerOverCurrentCommand;
+import frc.robot.commands.RumbleCommand;
+import frc.robot.control.AutonomousChooser;
+import frc.robot.control.InstalledHardware;
+import frc.robot.control.ManualInputInterfaces;
+import frc.robot.control.SubsystemCollection;
+import frc.robot.subsystems.DrivetrainPowerSubsystem;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import frc.robot.subsystems.EveryBotPickerSubsystem;
+import frc.robot.subsystems.PowerDistributionPanelWatcherSubsystem;
+import frc.robot.common.PortSpy;
+
+/**
+ * This class is where the bulk of the robot should be declared. Since
+ * Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in
+ * the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of
+ * the robot (including
+ * subsystems, commands, and button mappings) should be declared here.
+ */
+public class RobotContainer {
+
+  private SubsystemCollection subsystems = new SubsystemCollection();
+  private final AutonomousChooser autonomousChooser;
+
+  /**
+   * The container for the robot. Contains subsystems, OI devices, and commands.
+   */
+  public RobotContainer() {
+
+    // init the pdp watcher
+    this.initializePowerDistributionPanelWatcherSubsystem();
+
+    // init the various subsystems
+    this.initializeDrivetrainSubsystem();
+
+    // init the input system
+    this.initializeManualInputInterfaces();
+
+    // arm and picker later
+    this.initializeEveryBotPickerSubsystem();
```

Comment:
So the picker, is it a thing on this robot?

Reply:
Swapped mention of everybot picker to wrist subsystem